extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/JaegerPropagator.java

```diff
@@ -187,6 +208,42 @@ public static JaegerPropagator getInstance() {
     return buildSpanContext(traceId, spanId, flags);
   }
 
+  private static <C> Baggage getBaggageFromHeader(C carrier, Getter<C> getter) {
+    Baggage.Builder builder = null;
+
+    for (String key : getter.keys(carrier)) {
+      if (key.startsWith(BAGGAGE_PREFIX)) {
+        if (key.length() <= BAGGAGE_PREFIX.length()) {
+          continue;
+        }
+
+        if (builder == null) {
+          builder = Baggage.builder();
+        }
+        builder.put(key.substring(BAGGAGE_PREFIX.length()), getter.get(carrier, key));
+      } else if (key.equals(BAGGAGE_HEADER)) {
+        builder = parseBaggageHeader(getter.get(carrier, key), builder);
```

Comment:
Jaeger SDKs would combine baggage from `uberctx-{key}` and `jaeger-baggage` headers, you're overriding it instead.

Reply:
Actually they are getting merged, i.e. see [this test](https://github.com/open-telemetry/opentelemetry-java/pull/1549/files#diff-4e39f9e60626e3859be5734ea004b64bbbc1530e9a3d8e685122164aebfe27cdR423) - let me know otherwise :)

Reply:
L225 replaces the builder, so they are only merged if jaeger-baggage header is first in the list, as it is in the unit test, moving this header to the end of the list will ignore uberctx entries, as far as I can tell.

Reply:
Verified that it works correctly by putting HEADER at the end (of the Map), i.e. L225 takes any **existing** builder and uses it if already exists.